CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 # Changelog
 All notable changes to this project will be documented in this file. 
 
+## [2.0.x] - 2020-11-20
+
+### Changed
+- CKKS : API of evaluator.RotateColumns becomes Evaluator.Rotate
+- CKKS : the change of variable in evaluator.EvaluateCheby isn't done automatically anymore and the user must do it before calling the function to ensure correctness. The reason is that additional constant might be merged with the change of variable and/or the change of variable might be merged with previous operations. This leaves more freedom to the user to optimize a circuit. The method also doesn't take anymore into account the change of variable for checking if the number of remaining levels is sufficient to evaluate the polynomial.
```

Comment:
Please keep entries in the changelog short. What you describe here belongs to the godoc.

Reply:
reduced the change size